openedx/openedx-platform
# Revert "fix: Open Responses Styling and Layout (#33717)"

## Description

This reverts commit 1dab744ba2785ec220cc40d7d8a932393b46f23f introduced in https://github.com/openedx/edx-platform/pull/33717, which added ORA specific styles into the Platform's SCSS.

The fix is moved to ORA in https://github.com/openedx/edx-ora2/pull/2233.

Useful information to include:

- Impacts "Instructors"/"Staff"

Before:
![ora_before](https://github.com/user-attachments/assets/cf78d455-a3e9-41e9-980b-aa4ec26143a5)

After:
![ora_after](https://github.com/user-attachments/assets/3c21beaf-0016-4416-87b6-165047137ce8)


## Supporting information

Link to other information about the change, such as Jira issues, GitHub issues, or Discourse discussions.
Be sure to check they are publicly readable, or if not, repeat the information here.

## Testing instructions

* Checkout to PR branch
* Install the PR branch from https://github.com/openedx/edx-ora2/pull/2233
* Verify that the tables are clean and readable without any spacing issues affecting the text.

## Deadline

"None"

## Other information

Include anything else that will help reviewers and consumers understand the change.

- Does this change depend on other changes elsewhere?
- Any special concerns or limitations? For example: deprecations, migrations, security, or accessibility.
- If your [database migration](https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations) can't be rolled back easily.
